scripts/readmeFiles.js

```diff
@@ -65,7 +65,7 @@ const createPropsTable = ({ info }) => {
     let type = "-";
     if (typeKey in v) {
       if (v[typeKey].name === "union") {
-        type = `[${v[typeKey]["tsType" in v ? "elements" : "value"].map(i => i.name)}]`;
+        type = `[${v[typeKey]["tsType" in v ? "elements" : "value"]?.map(i => i.name)}]`;
```

Comment:
What does this fix?

Reply:
Running `generateReadme` throws a weird error (when no package.json is found for the package):
![Screenshot 2023-02-10 at 14 32 10](https://user-images.githubusercontent.com/117665459/218106950-95455545-cc2a-4091-8286-c8d8b59795a7.png)

It does not fix anything, hides the error 😄 It can be removed as well